### ISSUE TITLE
Loading Screen delay caused by Tidy Plates

### ISSUE THREAD

**Issue by SilverSaw** [ImogenBits/tidyplates#20]
Hello,

Issue:
When i enable Tidy Plates, Tidy Plates Hub and Tidy Plates Widgets my loading screen takes extra 10 sec of loading after every /reload or entering world anywhere. 

Test conditions:
I have like 50 addons and start disabling all of them one by one. I found that loading screen were fixed when i disabled tidy plates. Then i disabled all of my addons and enabled only Tidy Plates. I find out that world is still loading slowly sticking in loading screen for 10 more secs. When i enable all of my addons and disable only Tidy Plates loading screen is back to normal.

**Comment by ImogenBits** [comment]
Are you using ElvUI? If so, this is a known issue that will be fixed with the next release. If you want a fix right now you can download and manually install the version on the smallFixes branch, but that does currently have some UI scaling problems in its menus.

**Comment by SilverSaw** [comment]
No i am not using ElvUI.

**Comment by ImogenBits** [comment]
Are you using a custom UI scale (can be set in the blizzard options somewhere), that's the same issue?

**Comment by SilverSaw** [comment]
Yes i am using custom UI scale 0.95. I tried to turn it off and it is the same delay on reload only with tidy plates on.

**Comment by ImogenBits** [comment]
Try installing this version [here](https://github.com/ImogenBits/tidyplates/archive/smallFixes.zip).

**Comment by SilverSaw** [comment]
Installed and it is still the same, random delay between 7-10 sec

**Comment by mango201** [comment]
I have the same issue as above. I play wow in window more, and if i try and put on window maximise it lags the client for about 6-10 sec this only happens with tidy plates on without it it resize fine without any issues.

**Comment by ImogenBits** [comment]
I'm currently pretty busy with my studies. I and some others I've talked to also have absolutely no clue what is causing this issue. So I don't see a fix coming super shortly. I'll try to get it ready quickly after SL release, but almost definitely not before then.

**Comment by mango201** [comment]
hey again @ImogenBits  Appreciate what you're doing on the addon 👍,  I think I figured out a "manual fix" with the "lagging" when window resizing which might be helpful in trying to figure this out. So to remove this lag i had to open the tidyplates options and manually "open" each of the four profiles (Damage,Tank,Healer and gladiator.) Doing this removes the window lag. Also when i opened the settings tab just noticed how large the texts is. 
![image](https://user-images.githubusercontent.com/10214927/99864751-6e983880-2ba5-11eb-9dea-4a08d0cb211e.png)

**Comment by SilverSaw** [comment]
Mine interface is looking good, not any alignment issues like showed Damage Profile text. Also i tried to check all of the profile like mentioned, but there is no effect, still loading delay.

**Comment by SilverSaw** [comment]
Do you have any progress on this, same annoying loading delay still persist.
